### DISCUSSION TITLE
Help pls

### DISCUSSION THREAD

**Discussion by Hentina** [Kas-tle/java2bedrock.sh#1702] [Converter Help]
https://github.com/Kas-tle/java2bedrock.sh/actions/runs/3509068975/jobs/5877912214 
it fails

**Comment by Kas-tle** [comment]
Mediafire does not offer direct download links.

**Reply by Hentina** [reply]
where do i get a direct download link

**Reply by BobtheBob123456** [reply]
Use Dropbox
